### PR TITLE
Ft gratefuls journals single

### PR DESCRIPTION
**Pull request by Schade9**
## Description

_Adds endpoints for getting a single journal and grateful_

### CODE REVIEW

File: app/controllers/journal.py [hunk 1]
```diff
@@ -57,6 +56,25 @@ def get(self, user_id):
 
 
 class JournalsDetailView(Resource):
+    @jwt_required
+    def get(self, journal_id):
+        """
+        Getting a particular grateful
```
Comment (MutegekiHenry) [inline app/controllers/journal.py]: journal not grateful

File: app/controllers/journal.py [hunk 2]
```diff
@@ -57,6 +56,25 @@ def get(self, user_id):
...
+        """
+        journal_schema = JournalSchema(many=False)
+
+        journal = Journal.get_by_id(journal_id)
+
+        if not journal:
+            return dict(status='fail', message=f'No grateful data'), 404
```
Comment (MutegekiHenry) [inline app/controllers/journal.py]: *journal

File: app/routes/__init__.py [hunk 3]
```diff
@@ -53,7 +53,7 @@
 # Gratesfuls routes
 api.add_resource(GratefulsView, '/gratefuls', endpoint='grateful_records')
 api.add_resource(GratefulsView, '/gratefuls/<string:user_id>')
-api.add_resource(GratefulsDetailView, '/gratefuls/<string:grateful_id>')
+api.add_resource(GratefulsDetailView, '/grateful/<string:grateful_id>')
```
Comment (MutegekiHenry) [inline app/routes/__init__.py:56]: this has to be updated in the api-docs as well

File: app/routes/__init__.py [hunk 4]
```diff
@@ -68,4 +68,4 @@
 # Journal routes
 api.add_resource(JournalsView, '/journals', endpoint='journals_records')
 api.add_resource(JournalsView, '/journals/<string:user_id>')
-api.add_resource(JournalsDetailView, '/journals/<string:journal_id>')
\ No newline at end of file
+api.add_resource(JournalsDetailView, '/journal/<string:journal_id>')
```
Comment (MutegekiHenry) [inline app/routes/__init__.py:71]: this has to be updated in the api-docs as well

**Review by MutegekiHenry — CHANGES_REQUESTED** [review]
Great work, just a few minor fixes required

File: app/controllers/journal.py (hunk 1 above)
Comment (Schade9) [reply app/controllers/journal.py]: fixed

File: app/controllers/journal.py (hunk 2 above)
Comment (Schade9) [reply app/controllers/journal.py]: fixed

File: app/routes/__init__.py (hunk 3 above)
Comment (Schade9) [reply app/routes/__init__.py:56]: I changed all of them

**Review by MutegekiHenry — APPROVED** [review]
LGTM

**Pull request merged**